### PR TITLE
FontAwesome: bump awesomeness

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: media-fonts/fontawesome/fontawesome-5.0.6.recipe [hunk 1]
```diff
@@ -5,11 +5,11 @@ https://discuss.haiku-os.org uses it as well.
 As long as Haiku doesn't support loading a font that's not installed, you \
 should install this webfont to see all the icons on the discussion forum."
 HOMEPAGE="http://fontawesome.io"
```
Comment (alaviss) [inline media-fonts/fontawesome/fontawesome-5.0.6.recipe]:
```diff
-HOMEPAGE="http://fontawesome.io"
+HOMEPAGE="https://fontawesome.com/"

File: media-fonts/fontawesome/fontawesome-5.0.6.recipe [hunk 2]
```diff
@@ -5,11 +5,11 @@ https://discuss.haiku-os.org uses it as well.
...
-COPYRIGHT="2013-2016 Dave Gandy"
+COPYRIGHT="2013-2018 Dave Gandy"
 LICENSE="SIL Open Font License v1.1"
 REVISION="1"
-SOURCE_URI="https://github.com/FortAwesome/Font-Awesome/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="de512ba0e1dead382bbfce372cde74b3f18971d876fffb635ee9333f0db05d43"
+SOURCE_URI="https://use.fontawesome.com/releases/v5.0.6/fontawesome-free-$portVersion.zip"
```
Comment (alaviss) [inline media-fonts/fontawesome/fontawesome-5.0.6.recipe]:
```diff
-SOURCE_URI="https://use.fontawesome.com/releases/v5.0.6/fontawesome-free-$portVersion.zip"
+SOURCE_URI="https://use.fontawesome.com/releases/v$portVersion/fontawesome-free-$portVersion.zip"

File: media-fonts/fontawesome/fontawesome-4.7.0.recipe [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+As long as Haiku doesn't support loading a font that's not installed, you \
+should install this webfont to see all the icons on the discussion forum."
+HOMEPAGE="http://fontawesome.io"
+COPYRIGHT="2013-2016 Dave Gandy"
+LICENSE="SIL Open Font License v1.1"
+REVISION="1"
+SOURCE_URI="https://github.com/FortAwesome/Font-Awesome/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="de512ba0e1dead382bbfce372cde74b3f18971d876fffb635ee9333f0db05d43"
+SOURCE_DIR=""
+DISABLE_SOURCE_PACKAGE=yes
```
Comment (fbrosson) [inline media-fonts/fontawesome/fontawesome-4.7.0.recipe]: Suggestion (based on metrics :)
```diff
 SOURCE_DIR=""
-DISABLE_SOURCE_PACKAGE=yes
+
 ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE=yes
```

FWIW, we currently have 68 recipes with `DISABLE_SOURCE_PACKAGE` after `ARCHITECTURES`, and only 17 recipes with `DISABLE_SOURCE_PACKAGE` before `ARCHITECTURES`.

File: media-fonts/fontawesome/fontawesome-4.7.0.recipe [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+https://discuss.haiku-os.org uses it as well.
+As long as Haiku doesn't support loading a font that's not installed, you \
+should install this webfont to see all the icons on the discussion forum."
+HOMEPAGE="http://fontawesome.io"
+COPYRIGHT="2013-2016 Dave Gandy"
+LICENSE="SIL Open Font License v1.1"
+REVISION="1"
+SOURCE_URI="https://github.com/FortAwesome/Font-Awesome/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="de512ba0e1dead382bbfce372cde74b3f18971d876fffb635ee9333f0db05d43"
+SOURCE_DIR=""
```
Comment (fbrosson) [inline media-fonts/fontawesome/fontawesome-4.7.0.recipe]:
```diff
-SOURCE_DIR=""
+SOURCE_DIR="Font-Awesome-$portVersion"
```

File: media-fonts/fontawesome/fontawesome-4.7.0.recipe [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+BUILD_PREREQUIRES="
+	coreutils
+	"
+
+INSTALL()
+{
+	FONTDIR=$fontsDir/ttfonts
+	mkdir -p ${FONTDIR}
+	cd Font-Awesome-$portVersion
```
Comment (fbrosson) [inline media-fonts/fontawesome/fontawesome-4.7.0.recipe]:
```diff
INSTALL()
 {
 	FONTDIR=$fontsDir/ttfonts
 	mkdir -p ${FONTDIR}
-	cd Font-Awesome-$portVersion
 	cp -a fonts/fontawesome-webfont.ttf ${FONTDIR}
 }
```

File: media-fonts/fontawesome/fontawesome-5.0.6.recipe [hunk 6]
```diff
@@ -4,11 +4,11 @@ the internet to depict certain actions. Haiku's discussion forum at \
...
 should install this webfont to see all the icons on the discussion forum."
-HOMEPAGE="http://fontawesome.io"
+HOMEPAGE="https://fontawesome.com/"
 COPYRIGHT="2013-2018 Dave Gandy"
 LICENSE="SIL Open Font License v1.1"
 REVISION="1"
-SOURCE_URI="https://use.fontawesome.com/releases/v5.0.6/fontawesome-free-$portVersion.zip"
+SOURCE_URI="https://use.fontawesome.com/releases/v$portVersion/fontawesome-free-$portVersion.zip"
 CHECKSUM_SHA256="6a4be82b5ebfa4c8afd8d88ac6b0a9092efd7bfcc7f7affb03e051c9c26eddaf"
 SOURCE_DIR=""
```
Comment (fbrosson) [inline media-fonts/fontawesome/fontawesome-5.0.6.recipe]:
```diff
-SOURCE_DIR=""
+SOURCE_DIR="fontawesome-free-$portVersion"
```

Later:
```diff
 INSTALL()
 {
 	FONTDIR=$fontsDir/ttfonts
 	mkdir -p ${FONTDIR}
-	cd fontawesome-free-$portVersion
 	cp -a on-server/webfonts/*.ttf ${FONTDIR}
 }
```

**Pull request merged**